packages/sync-actions/src/product-types-actions.js

```diff
@@ -15,16 +15,19 @@ const REGEX_UNDERSCORE_NUMBER = new RegExp(/^_\d+$/)
 const getIsChangedOperation = key => REGEX_NUMBER.test(key)
 const getIsRemovedOperation = key => REGEX_UNDERSCORE_NUMBER.test(key)
 
-export const baseActionsList = [
+export const updateActionsDefinitions = [
+  // required
   { action: 'changeName', key: 'name' },
-  { action: 'setKey', key: 'key' },
   { action: 'changeDescription', key: 'description' },
+  // optional
+  { action: 'setKey', key: 'key' },
 ]
 
 export function actionsMapBase(diff, previous, next) {
   return buildBaseAttributesActions({
     diff,
-    actions: baseActionsList,
+    actions: updateActionsDefinitions,
```

Comment:
this was a renaming to help myself to make sense of `buildBaseAttributesActions` 😄 
I'll keep it if you don't mind